vignettes/wwinference.Rmd

```diff
@@ -360,7 +374,21 @@ state-level R(t) estimate.
 We can generate this directly on the output of `wwinference()` using:
 ```{r extracting-draws}
 draws_df <- get_draws_df(ww_fit)
+
+# Print the names of the variables included in the output dataframe
+cat(
+  "Variables in dataframe: ",
+  sprintf("%s", paste(unique(draws_df$name), collapse = ", "))
+)
 ```
+Note that by default the `get_draws_df()` function will return  a tidy long
+dataframe with all of the posterior draws joined to applicable data for each of
+the included variables. If only interested in the hospital admissions estimates
+(`"predicted counts"`) or the wastewater estimates (`"predicted wastewater"`),
+we suggest immediately filtering to `name %in% c(your_desired_outputs)`
+so as not to carry around extra data.
+This is particularly important if there are a large
+number of wastewater sites, as this dataframe can get quite large.
```

Comment:
```suggestion
the included variables. To examine a particular variable (e.g. `"predicted counts"` for posterior 
predicted hospital admissions), filter the data frame based on the `name` column.
```

Related but out of scope questions:
1. should that column be named "name" and not "variable_name"?
2. should variable names really have spaces?

Reply:
Or maybe these shouldn't be called variables, but something more general like output type? I think it's nice for something like R(t) not to be `subpop_r_t`.

Reply:
One reasonable approach there is to have one column with a verbose human-readable name and another with a shorter, no-spaces internal name. One can then do filtering mainly on the latter column

Reply:
I think that's a good idea. I might just put that in this PR since its top of mind